### ISSUE TITLE
Direct download links broken

### ISSUE THREAD

**Issue by DragoonAethis** [autyzm-pg/friendly-plans#49]
Direct download links on http://autyzm.eti.pg.gda.pl/przyjaznyplan/instalacja/ (actually Dropbox links, but close enough) no longer work (404). Release APKs should be reuploaded.

**Comment by malsolec** [comment]
Good point! Thanks. The link should be replaced with: https://play.google.com/store/apps/details?id=com.przyjaznydamianek&hl=pl

**Comment by DragoonAethis** [comment]
I'd just reupload them, eg on GitHub (as a release) and link to the releases page, no more site updates and stays always up-to-date. Also, Chinese tablets without Google Play would benefit from that, too.

**Comment by mikolevy** [comment]
Good idea. I found apk version of app on CD from the university. I'll check is it the right version and then I'll upload it to GitHub.
We can't use apk generated from current repo version because of:
https://github.com/autyzm-pg/friendly-plans/issues/53#issuecomment-261650462
